boards/seeeduino_xiao/board.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C)  2021 Franz Freitag, Justus Krebs, Nick Weiler
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_seeeduino_xiao
+ * @{
+ *
+ * @file
+ * @brief       Board specific implementations for the Seeeduino XIAO board
+ *
+ * @author      Franz Freitag <franz.freitag@st.ovgu.de>
+ * @author	Justus Krebs <justus.krebs@st.ovgu.de>
+ * @author	Nick Weiler <nick.weiler@st.ovgu.de>
+ * @author	Benjamin Valentin <benpicco@googlemail.com>
```

Comment:
I don't even have this board :wink:

Reply:
We deleted you as author from every code file.